Yoast/javascript
# Add Arabic passive voice assessment

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds passive voice assessment for Arabic, including a check, a number or specs and a list of Arabic passive verbs with long vowel in their root. 

* [Yoast Seo Free] Adds an assessment for the recognition of the passive voice in Arabic texts (for example: `أُنشئت الشركة في الثمانينيات` or `ودعيت هذه المجموعات للإدلاء بملاحظات ختامية في الجلسة الأخيرة`).

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all tests still pass.

* In the content-analysis app:

* Set the locale to ar;
* Paste this text: 
`إنها تحمل الكتاب إلى الفصل. كما أنها تحضر حقيبة الظهر وزجاجة الماء الخاصة بها. قالت إنها لا تحب أن تكون في الفصل.
هناك الكثير من عيون الحكم.`  in the Text input window (it won't detect any passives);
* Add these sentences containing Arabic passive verb forms: 
`فُرش البيت الجديد.
حماسي في تلك الأيام أثير كلياً بواسطة الكحول.`
The passive voice assessment should read `25% of the sentences contain passive voice, which is more than the recommended maximum of 10%. Try to use their active counterparts.`;
* Add two more sentences using passive verb with a diacritic damma `وُضع` and a verb from the `passiveVerbsWithLongVowel` list. The passive voice assessment should read `Passive voice: 33.3% of the sentences contain passive voice, which is more than the recommended maximum of 10%. Try to use their active counterparts.`

## QA Test Instructions
 * Set the WP environment language to Arabic
 * Test this functionality in Wordpress SEO Free
 * You should see a bullet assessment for passive verb forms in the Readability section 
 * Add these sentences to your post: 
  `فُرش البيت الجديد.`
`حماسي في تلك الأيام أثير كلياً بواسطة الكحول.`
`لربّما هو سوعد.`
 * Check that the passive voice bullet turns red


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-522
